src/core/alien/kernels/simple_csr/redistribution/SimpleCSRDistributorImpl.h

```diff
@@ -304,6 +309,7 @@ void SimpleCSRDistributor::distribute(const SimpleCSRVector<NumT>& src, SimpleCS
 template <typename T>
 void SimpleCSRDistributor::_resizeBuffers(const int bb)
 {
+  // comm_info should be templated by the type to avoid cast and explicit size computations
```

Comment:
No because we want to  use the same buffers for different litteral types

Reply:
Ok, thanks for the explanation.